cmd/eve.go

```diff
@@ -209,8 +211,6 @@ func newSshEveCmd(cfg *openevec.EdenSetupArgs) *cobra.Command {
 	}
 
 	sshEveCmd.Flags().StringVarP(&cfg.Eden.SSHKey, "ssh-key", "", filepath.Join(currentPath, defaults.DefaultCertsDist, "id_rsa"), "file to use for ssh access")
-	sshEveCmd.Flags().StringVarP(&cfg.Runtime.Host, "eve-host", "", defaults.DefaultEVEHost, "IP of eve")
```

Comment:
we don't use this parameter in function. I'm not sure wether it's correct behaviour @giggsoff

Reply:
Seems you are right, but it is quite strange.